src/wrapt/wrapt.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import Any, Generic, TypeVar, Callable, Optional
+
+F = TypeVar('F', bound=Callable[..., Any])
+A = TypeVar('A', bound=Callable[..., Any])
+T = TypeVar("T", bound=Any)
+
+def decorator(wrapper: F, enabled: Optional[bool] = None, adapter: Optional[A] = None) -> F: ...
+
+class ObjectProxy(Generic[T]):
```

Comment:
I think that pyi files should also follow pep-8, which means 2 blank lines between functions and classes declared at the module scope.

Maybe just run `ruff` or `black` to auto-format this file?